pkg/redis/client_test.go

```diff
@@ -155,6 +155,17 @@ func (s *ClientWithMiniRedisTestSuite) TestHMGet() {
 	s.Equal([]interface{}{nil, "1"}, vals, "there should be no error on HSet")
 }
 
+func (s *ClientWithMiniRedisTestSuite) TestHGetAll() {
+	err := s.client.HSet(context.Background(), "key", "field", "value1")
+	s.NoError(err, "there should be no error on HSet")
+	serr := s.client.HSet(context.Background(), "key", "field2", "value2")
+	s.NoError(serr, "there should be no error on HSet")
+
+	vals, err := s.client.HGetAll(context.Background(), "key")
+	s.NoError(err, "there should be no error on HgetAll")
+	s.Equal(map[string]string{"field": "value1", "field2": "value2"}, vals, "there should be no error on HSet")
```

Comment:
You're message is misleading, as the assert does not have anything to do with `errors on HSet`.
You can either fix it or remove it.
For most assertions, we don't provide error messages, as they are just duplication the assertion itself:
e.g. s.NoError(error, "there should be no error") contains the same information twice.